spec/jobs/bgs/submit_form686c_job_spec.rb

```diff
@@ -3,20 +3,26 @@
 require 'rails_helper'
 
 RSpec.describe BGS::SubmitForm686cJob, type: :job do
-  subject { described_class.new.perform(user.uuid, dependency_claim.id, vet_info) }
+  subject { described_class.new.perform(user.uuid, user.icn, dependency_claim.id, vet_info) }
 
   let(:user) { FactoryBot.create(:evss_user, :loa3) }
   let(:dependency_claim) { create(:dependency_claim) }
   let(:all_flows_payload) { FactoryBot.build(:form_686c_674_kitchen_sink) }
+  let(:birth_date) { '1809-02-12' }
   let(:vet_info) do
     {
       'veteran_information' => {
-        'birth_date' => '1809-02-12',
         'full_name' => {
-          'first' => 'WESLEY', 'last' => 'FORD', 'middle' => nil
+          'first' => 'WESLEY', 'middle' => nil, 'last' => 'FORD'
```

Comment:
Same note as above re: consider using `Faker`

Reply:
for this I went off of precedent in the file, could look at Faker in the future.